File: all/abc186/abc186_b/abc186_b.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
  int h, w, ans = 0; cin >> h >> w;
  vector<int> a(h*w);
  for(int i = 0; i < h*w; i++) cin >> a[i];
  sort(a.begin(), a.end());
  int min = a[0];
  for(int i = 1; i < h*w; i++) ans += a[i] - min;
  cout << ans;
}

File: all/abc186/abc186_b/abc186_b.py
h, w = map(int, input().split())
a = []
for _ in range(h):
  a.append(list(map(int, input().split())))

mi = 1e9
for i in range(h):
  for j in range(w):
    mi = min(mi, a[i][j])

ans = 0
for i in range(h):
  for j in range(w):
    ans += a[i][j] - mi

print(ans)